draft-yasskin-httpbis-origin-signed-exchanges-impl.md

```diff
@@ -1044,6 +1088,28 @@ The other fields are the same as the registration in
 
 # Change Log
 
+draft-03
+
+Vs. draft-02
```

Comment:
Should header-structure 07 -> 09 be in this changelog somewhere?

Reply:
It's covered in "Updates to match {{I-D.yasskin-http-origin-signed-responses-05}}."